src/actions_manager.coffee

```diff
@@ -89,4 +89,15 @@ define [
 
       payload
 
+    _buildMetadata: (sbm) ->
+      meta = @session.metadata
+      return (meta || '') unless sbm
+
+      if meta
+        @session.metadata.tags = if meta.tags then meta.tags + ',sbm' else 'sbm'
```

Comment:
I was thinking about this oprations
```
meta.tags + ',sbm'
```

Is there any chance "meta.tags" to be defined but empty string?
In such a case the result would be 
```
meta.tags == ',sbm'
```

This tags value could be a problem in analytics server?

Reply:
I think coffeescript evaluates empty strings as false in these cases

![image](https://user-images.githubusercontent.com/26133545/188610000-78f6c4f8-aa08-4b98-92cc-e09109041f46.png)

Reply:
```
if meta.tags
```
seems to check against undefined, empty, null values, correct?

Reply:
yes that's right